### DISCUSSION TITLE
No connecting passageway between biomes

### DISCUSSION THREAD

**Discussion by HawkinsThe71st** [FakeFishGames/Barotrauma#14653] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Using the map seed `XJ64wtek`, there is no connection between cold caverns and the europan ridge:
![image](https://github.com/user-attachments/assets/9262b4c6-69c4-45c3-aabc-a2d726a7f606)
Save file:
[broken seed test.zip](https://github.com/user-attachments/files/17042353/broken.seed.test.zip)

### Reproduction steps

1. Create new save with seed XJ64wtek

### Bug prevalence

Happens regularly

### Single player or multiplayer?

Single player

### Version

v1.5.9.2 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Duplicate of https://github.com/FakeFishGames/Barotrauma/discussions/13791. Should be fixed in the current unstable version, and the fix will be included in the next update. Sorry for the inconvenience!
